stacks-signer/src/runloop.rs

```diff
@@ -412,7 +412,7 @@ impl<Signer: SignerTrait<T>, T: StacksMessageCodec + Clone + Send + Debug>
         if self.state == State::NoRegisteredSigners {
             let next_reward_cycle = current_reward_cycle.saturating_add(1);
             if let Some(event) = event {
-                info!("Signer is not registered for the current reward cycle ({current_reward_cycle}) or next reward cycle ({next_reward_cycle}). Waiting for confirmed registration...");
```

Comment:
Hate to nitpick here, but I think maybe just remove the "Waiting for confirmation..." but then all good :)

Reply:
Okay 👍

Reply:
Done